packages/obonode/obojobo-sections-assessment/server/assessment.js

```diff
@@ -275,6 +275,19 @@ class Assessment extends DraftNode {
 		)
 	}
 
+	static fetchAttemptByIdAndUserId(attemptId, userId) {
```

Comment:
added for now just to avoid having to re-write `getAttempt`.
1.  prefer fetch over get for models
2. using the same convention used in the other models for arguments and intent